### ISSUE TITLE
Lab 06

### ISSUE THREAD

**Issue by mtwelker** [Watts-College/cpp-529-fall-2021#12]
Prof. Howell, @AntJam-Howell 
It looks like Parts 1 and 2 of Lab 06 are exactly what we did last week in Lab 05 (if we don't want to select new variables).  Am I understanding that right?  And if so, can we just re-use what we did in Lab 05 for those parts?
Thank you,
Michelle Welker

**Comment by AntJam-Howell** [comment]
@mtwelker Yes, if you do not select any new variables, you are free to repeat code chunks wherever the questions overlap. 

Best,
